### ISSUE TITLE
toStringTag

### ISSUE THREAD

**Issue by stevenvachon** [jsdom/whatwg-url#91]
```js
URL.prototype[Symbol.toStringTag] = "URL"
```

**Comment by domenic** [comment]
I can't tell what bug this is reporting, so closing...

**Comment by stevenvachon** [comment]
You could've asked. Now, please reopen.

This library does not make use of toStringTag:
```js
new Map()[Symbol.toStringTag]
//-> 'Map'

new (require('url').URL)('http://host/')[Symbol.toStringTag]
//-> 'URL'

new (require('whatwg-url').URL)('http://host/')[Symbol.toStringTag]
//-> undefined
```

**Comment by TimothyGu** [comment]
This has been fixed on the webidl2js side: https://github.com/jsdom/webidl2js/commit/0590e5a8b3e91edf39d40209795ff41b87787bfe

It's only a matter of time before we update to a new version of webidl2js.

**Comment by stevenvachon** [comment]
@TimothyGu thank you.

**Comment by TimothyGu** [comment]
FYI this has been properly fixed in [v6.0.0](https://github.com/jsdom/whatwg-url/releases/tag/v6.0.0).
